refresh/index.js

```diff
@@ -48,7 +114,90 @@ module.exports = generators.Base.extend({
       }
     },
 
+    createBasicProject: function() {
```

Comment:
This function should probably be moved into the `writing` phase (along with any others that write files).